server/util/clickhouse/schema/schema.go

```diff
@@ -226,6 +226,10 @@ type Execution struct {
 
 	Experiments []string `gorm:"type:Array(LowCardinality(String))"`
 
+	// If an executionID is retried, the RunID will differ for each retry attempt.
+	// Can be unset for executions that aren't retried.
+	RunID string
```

Comment:
I think ideally each execution ID would only be used once (representing a single "execution attempt"), and then we wouldn't need this field. Since this is an existing limitation of how the scheduler can internally retry executions, I think this change is fine / good, but it feels like yet another "wart" that we are incurring by having the scheduler treat a "workflow job" / "workflow run" the same as a "workflow execution," rather than modeling them as 2 separate concepts. Related: https://github.com/buildbuddy-io/buildbuddy-internal/issues/4924